src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
when your destruction haves more than 2 args split it in rows

Reply:
Lint says that is ok. Should I ignore it?

Reply:
It mostly depends on a project, as a rule of thumb - whenever a line goes too long we should split it in rows, it makes even more sense when you want to extract properties on the props level: 

```
export const Person = ({ 
    person: {
         name,...
    } 
}) => {
```

It just makes your code more readable and it's easier to find / modify something in props. 
Even if linter still don't mark it as an error, it still makes sense to write arguments in a row.

It's not a mistake here in terms of this assignment, but in 90% your reviewer at work will ask you for these changes, so it's a good idea to develop this code-styling behaviour.